mkdocs.yml

```diff
@@ -43,8 +43,10 @@ nav:
       - products.md
       - Digital Elevation Models: dems.md
       - Product guides:
+          - Introduction to SAR: guides/sar_intro.md
```

Comment:
I'd prefer to see the markdown named `introduction_to_sar.md` to match the page title and ToC entry, but it's not a blocking issue

Reply:
That's fine with me.